.github/ISSUE_TEMPLATE/build_error.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Build Error
+about: Create a detailed report about an error during in the BitShares Core build process.
+
+---
+
+**Instructions**
+Please include a detailed Title above. Next, please complete the following sections below:
+* Build Error
+* Build Environment
+* Steps To Reproduce
+* Screenshots (optional)
```

Comment:
No screenshots, rather a *full* console log including all commands entered and their output.